config/orchestrations/collection-operator/0.1/collection-operator.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: kabanero-collection-operator
+spec:
+  selector:
+    app: kabanero-collection-operator
+  ports:
+  - protocol: TCP
+    port: 443
+    targetPort: 9443
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: kabanero-collection-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - "get"
+  - "create"
+- apiGroups:
+  - apps
+  resources:
+  - deployments/finalizers
+  resourceNames:
+  - collection-operator
+  verbs:
+  - "update"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - replicasets
+  - deployments
+  verbs:
+  - get
+- apiGroups:
+  - tekton.dev
+  resources:
+  - pipelines
+  - tasks
+  verbs:
+  - delete
+  - get
+  - create
+  - list
+  - update
+  - watch
+- apiGroups:
+  - kabanero.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kabanero-collection-operator
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kabanero-collection-operator
+subjects:
+- kind: ServiceAccount
+  name: kabanero-collection-operator
+roleRef:
+  kind: Role
+  name: kabanero-collection-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kabanero-collection-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kabanero-collection-operator
+  template:
+    metadata:
+      labels:
+        app: kabanero-collection-operator
+    spec:
+      serviceAccountName: kabanero-collection-operator
+      containers:
+        - name: kabanero-collection-operator
+          image: {{ .image }}
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
```

Comment:
Most of these environment variables will be un-necessary.  You can re-use WATCH_NAMESPACE to tell the controller what namespace it should watch for `kind: Collection`.